crates/flipperzero/src/subghz/error.rs

```diff
@@ -0,0 +1,22 @@
+use core::{error::Error, fmt::Display};
+
+#[derive(Clone, Copy, Debug)]
```

Comment:
`uDebug` should also be implemented when `Debug` is implemented.

Reply:
I didn't realize this, added with 1ebd7ca9132f77ced2fcc554628f39cdac56c5d6